contribs/gmf/src/print/component.js

```diff
@@ -176,6 +176,14 @@ exports.component_ = {
 
 exports.component('gmfPrint', exports.component_);
 
+/**
+ * @typedef {{
+ *     bbox: (boolean|undefined),
```

Comment:
Can we name it `haveBbox` or `useBbox` ? or something like that ?
(Or it's too long to change ?)

Reply:
Done :-)